### PR TITLE
Make sure keys gets arguments right

### PR DESCRIPTION
**Pull request by imurchie**
We get an array sometimes. Make sure we send a string along.

See https://github.com/appium/appium/issues/6441

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -116,6 +116,12 @@ commands.keys = async function (keys) {
     }
     await this.setValue(keys, el.ELEMENT);
   } else {
+    if (_.isArray(keys)) {
+      keys = keys.join('');
+    }
+    if (!_.isString(keys)) {
+      keys = `${keys}`;
```
Comment (jlipps) [inline lib/commands/general.js]: is this the same as `keys = keys.toString()`?

Comment (imurchie) [inline lib/commands/general.js]: It is, as long as `keys` is an array with a string inside. I think the spec says it ought to take an array of characters, in which case `['a', 'b', 'c'].toString() == 'a,b,c'`

Comment (jlipps) [inline lib/commands/general.js]: Hmm, I was more commenting on what we would prefer to standardize on if

    `${keys}`

and

    keys.toString()

are equivalent.

Comment (imurchie) [inline lib/commands/general.js]: Ah! I see. Yes, `toString` is better. And not sure why I went with the other, to be perfectly honest.

Comment (imurchie) [inline lib/commands/general.js]: https://github.com/appium/appium-ios-driver/pull/137

**Pull request merged**